vedanshujain/woocommerce
# Load API manually as its now merged into core

### All Submissions:






### Changes proposed in this Pull Request:

This PR enables loading of REST API using appropriate hooks in WooCommerce code so that it's available. This includes:

1. Removing wc-rest-api package from the composer.
2. Add API path to class-map configuration for autoloading. 

Also includes some code standard and minor fixes to unit tests.


### How to test the changes in this Pull Request:

1. Checkout this branch and wc-rest-api plugin (https://github.com/woocommerce/woocommerce-rest-api/). Disable the REST API plugin for now. Run `composer update`.
2. Go to WooCommerce > Status page and look for `WooCommerce REST API package` setting string. Make sure that its equal to current WC version.
3. Enable the REST API plugin and refresh the status page. It should now show the API Package version since that;s now loaded instead of API code.

### Other information:




### Changelog entry

> Dev - Merge API Package into core.
